tests/integration/targets/azure_rm_hdinsightcluster/aliases

```diff
@@ -0,0 +1,4 @@
+cloud/azure
+destructive
+shippable/azure/group1
+unstable # test is slow (~30 minute run time), not unstable, but this is better than unsupported
```

Comment:
is it ok to add comments in alias?

Reply:
that's added by mattclay, I am just merging his changes